### ISSUE TITLE
Proposal: Returning a message to docker client after powerstrip fails

### ISSUE THREAD

**Issue by aanm** [ClusterHQ/powerstrip#79]
Powerstrips should return a message to the docker client that made the request informing the request couldn't be completed if one of the adapters fail.
I propose that message to be showed would be a string from the adapter that would be present in ModifiedClientRequest.Body

**Comment by binocarlos** [comment]
Agreed - a message and a 5xx status code should end up at the docker client.  I think we should send the message back in the same format that the normal docker server sends its errors.  

ModifiedClientRequest.Body is for normal operations - I think if there is an error, a 500 status and a string that is however the docker server formats its errors - JSON or plaintext?

**Comment by aanm** [comment]
IMHO since the error is returned in a HTTP form, the error message it should be plainttext
